.github/workflows/build.yaml

```
@@ -66,3 +66,45 @@ jobs:
       node_type: "gpu-v100-latest-1"
       run_script: "ci/build_docs.sh"
       sha: ${{ inputs.sha }}
+  wheel-build-cpp:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-24.06
+    with:
+      matrix_filter: group_by([.ARCH, (.CUDA_VER|split(".")|map(tonumber)|.[0])]) | map(max_by(.PY_VER|split(".")|map(tonumber)))
+      build_type: ${{ inputs.build_type || 'branch' }}
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      script: ci/build_wheel_cpp.sh
+  wheel-build-python:
+    needs: wheel-build-cpp
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-24.06
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      script: ci/build_wheel_python.sh
+  wheel-publish-cpp:
+    needs: wheel-build-cpp
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-publish.yaml@branch-24.06
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      branch: ${{ inputs.branch }}
+      sha: ${{ inputs.sha }}
+      date: ${{ inputs.date }}
+      package-name: kvikio
```

This needs to match what's being uploaded in `ci/build_wheel_cpp.sh`.
```suggestion
      package-name: libkvikio
```

---

I wondered about that, but saw that rmm didn't do this: https://github.com/rapidsai/rmm/blob/branch-24.06/.github/workflows/build.yaml#L98

So maybe rmm needs a quick fix, too?

---

It looks like in rmm [the name used for the upload](https://github.com/rapidsai/rmm/blob/branch-24.06/ci/build_wheel_cpp.sh#L32) is [not the same as the package name](https://github.com/rapidsai/rmm/blob/branch-24.06/ci/build_wheel_cpp.sh#L6). It would have been nice to align, but it doesn't really matter since https://github.com/rapidsai/build-planning/issues/44 will make question superfluous.